src/App.jsx

```diff
@@ -21,6 +21,29 @@ const App = () => {
     return response;
   }
 
+  // const core = require('@actions/core');
+  // const github = require('@actions/github');
+  // const { context } = require('@actions/github')
```

Comment:
The initial state with `initDimensions` avoids having an undefined state on the server side. This effectively prevents  potential issues during server-side rendering (SSR). It also simplifies initial rendering.